src/server.py

```diff
@@ -395,15 +399,8 @@ def run_create_jobtypes_query():
             types = []
             for testtype in cursor.fetchall():
                 testtype = testtype[0]
-                # ignore talos, builds, jetpack
-                if testtype in ['svgr', 'svgr-e10s', 'svgr-osx', 'svgr-osx-e10s',
-                                'other', 'other-e10s', 'other-osx', 'other-osx-e10s',
-                                'chromez', 'chromez-e10s', 'chromez-osx', 'chromez-osx-e10s',
-                                'tp5o', 'tp5o-e10s', 'dromaeojs', 'dromaeojs-e10s',
-                                'g1', 'g2', 'g2-e10s', 'g1-e10s', 'g1-snow', 'g1-osx-e10s',
-                                'other_nol64', 'other_nol64-e10s', 'other_l64', 'other_l64-e10s',
-                                'xperf', 'xperf-e10s', 'other-e10s', 'dep', 'nightly', 'jetpack',
-                                'non-unified', 'valgrind']:
+                # ignore builds, jetpack
+                if testtype in ['dep', 'nightly', 'jetpack', 'non-unified', 'valgrind']:
                     continue
```

Comment:
I see you have removed talos jobs from consideration, do we want SETA active for talos jobs?